styles/simple.scss

```diff
@@ -1,18 +1,136 @@
 .n-newsletter-signup--simple {
-	display: block;
+	display: flex;
+	flex-direction: row;
+	flex-shrink: 0;
 	border: 0;
+	gap: 20px;
+	padding-top: 20px;
+	background-color: oColorsByName("white-80");
+
+	@include oGridRespondTo($from: S, $until: M) {
+		padding-left: 24px;
+		padding-bottom: 0;
+	}
+
+	@include oGridRespondTo($from: M) {
+		padding-left: 44px;
+		align-items: center;
+		padding-bottom: 20px;
+	}
+
+	.n-newsletter-signup__card_content {
+		display: flex;
+		padding-left: 0;
+
+		@include oGridRespondTo($from: S, $until: M) {
+			flex-direction: column;
+			align-items: start;
+		}
+
+		@include oGridRespondTo($from: M) {
+			flex-direction: row;
+			align-items: center;
+
+		}
+
+		h1 {
+			padding-left: 0;
+		}
```

Comment:
Good catch. The style mentioned above shouldn't be there since I switched to using an `<h2>` tag in the newsletter section. I'll remove it from the styles.